### PR TITLE
Feature/sc 138 charging history admin

### PR DESCRIPTION
**Pull request by mkajic20**
Added page for charging history for admin. The page displays all charges of all users. It uses search and pagination while displaying the data. Details about this task can be found on the [Confluence page](https://mkajic20.atlassian.net/browse/SC-138?atlOrigin=eyJpIjoiNzNhYzUwZTdmZDk2NDY0YTg2YzY4Yzk5NDAzMTc4ZGMiLCJwIjoiaiJ9)

### CODE REVIEW

File: Software/smart-charger/src/utils/api/adminHistory.js [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+  //         email: "mkajic20@foi.hr",
+  //       },
+  //       id: 2,
+  //     },
+  //   ],
+  //   success: true,
+  //   message: "List of all users' events",
+  //   page: 1,
+  //   totalPages: 1,
+  // };
```
Comment (irosic20) [inline Software/smart-charger/src/utils/api/adminHistory.js]: You should remove unnecessary comments.

**Review by irosic20 — CHANGES_REQUESTED** [review]
Everything looks good expect commented code in adminHistory.js file.

File: Software/smart-charger/src/utils/api/adminHistory.js (hunk 1 above)
Comment (mkajic20) [reply Software/smart-charger/src/utils/api/adminHistory.js]: Thank you, I removed the comments from the file.

**Review by irosic20 — APPROVED** [review]
Everything seems good now. Keep up the good work!

**Review by tsisko20 — APPROVED** [review]
Everything is good.

**Pull request merged**